README.md

```diff
@@ -49,6 +49,9 @@ To run cucumber tests only:
 ```sh
 ./gradlew consoleLauncherTest
 ```
+# Swagger
+## Local Url
+http://localhost:8080/swagger-ui/index.html
 
 Starting the app - TODO
```

Comment:
@patelila How do I start the app locally, so that I can see the Swagger API?

Reply:
Run CfeCrimeApplication

Reply:
> Run CfeCrimeApplication.java

@patelila could you tell me exactly what to type? I'm not familiar with running Java Spring apps, and although I could try and figure it out, it would be helpful to have the exact command known, or even better in the README

Reply:
java -jar ./build/libs/cfe-crime.jar

Reply:
Great, thanks for adding this to the readme